spec/observables/forkJoin-spec.ts

```diff
@@ -245,6 +274,15 @@ describe('Observable.forkJoin', () => {
     expectObservable(e1).toBe(expected);
   });
 
+  it('should raise error when any of source raises error with source that never completes', () => {
+    const e1 = Observable.forkJoin(
+      hot('------#'),
+      hot('----------'));
+    const expected = '------#';
+
```

Comment:
Can we vertically align these marble strings? It's good to make the next developer see the visual alignment of the events.